### PR TITLE
Feature/pay 1539 update android pay documentation

### PR DESCRIPTION
**Pull request by svenresch**

### CODE REVIEW

**Comment by aengusbates** [comment]
Let's align to the syntax of the tech world and go with "APIs", rather than "API's" or "A.P.I.'s". A quick Google suggests that the industry standard in tech is "APIs" - Google, Twitter, etc. 

Side note:  NY Times says that we would use an apostrophe if we were to say  "A.P.I.'s", but not "APIs"
https://afterdeadline.blogs.nytimes.com/2010/04/13/faqs-on-style/

**Comment by mikefay** [comment]
So, this may be a bit of a moot point, but we follow the Boston Manual for most everything, which the NY Time does not. Because we deal with things like industry jargon, we follow the style guide as laid out or approved by Joe in Stockholm. The decision was "API's", and we're beholden to it. We can take it up with Joe, but decision making on our end isn't going to go anywhere great.

File: source/docs/guides/android_pay/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: Android Pay
+layout: tutorial
+
+summary: >
+    Our Payments API now exposes a new 'android_pay' payment method and associated parameters to accept a base 64 encoded Android Pay Payment Token.
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: lowercase payment token

File: source/docs/guides/android_pay/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  footer: na.tocs.na_nav_footer
+  toc: false
+  header_active: Guides
+---
+
+# Android Pay
+
+## About Android Pay
+
+Our Payments API allows your mobile apps and online stores to accept payments using an Android device or, from a desktop, with the Google Chome web browser.
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: should API be APIs?

Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: remove commas

File: source/docs/guides/android_pay/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+When your customers use Android Pay, actual credit or debit card numbers are not sent with the payment. Instead a virtual account number is used to represent account info – so customer card details stay safe and secure.
+
+All of the directions and code samples to enable Android Pay in your app are available through Google documentation. Visit [Android Pay](https://www.android.com/pay/) to goes over the basics of Android Pay. [Android Pay for Developers](https://developers.google.com/android-pay/) has instructions on how to enable Android Pay and on how to create Android Pay payment tokens.
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: ..to goes.. --> ...to go...

File: source/docs/guides/android_pay/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+## Getting Started
+
+If this is your first time implementing our APIs we recommend reviewing our [Reference guide](/docs/references/payment_APIs/), to get familiar with Bambora's Payment APIs.
+
+You can find more about Bambora and Android Pay on [Github](https://github.com/bambora/na-payments-api-demo)
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: End with period?

File: source/docs/guides/android_pay/index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+## Requirements
+
+### Registering Your Android Pay Merchant ID
+
+Before you accept Android Pay with Bambora, you need to register an Android Pay Merchant ID to create and be able to access associated encryption keys.
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: Should Merchant ID be capitalized?

File: source/docs/guides/android_pay/index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+### Enable Android Pay
+
+To turn on Android Pay for your account, log into the [Member Area](https://web.na.bambora.com). Under **configuration**, click on **mobile payments**. From the Mobile Payments screen, you can enable Android Pay by creating a Android Pay Merchant ID for your mobile app.
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: ...an Android...

File: source/docs/guides/android_pay/index.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+<img src="/docs/guides/android_pay/android-mobile-payments-screenshot.png" alt="mobile-payments-screenshot">
+
+Click **ADD ANDROID MERCHANT ID** in order to create your *Android Pay Merchant ID*. Select to Copy the newly created Public Key and use it as as the publicKey parameter in your Android app.
```
Comment (davidlight77) [inline source/docs/guides/android_pay/index.md]: Public Key capitalized?

File: source/docs/guides/apple_pay/index.md [hunk 8]
```diff
@@ -16,13 +16,13 @@ navigation:
 
 ## About Apple Pay
 
-Our Apple Pay API allows your mobile app and online store to accept payments using Apple Watch, iPhone, iPad, or Safari.
+Our Payments API allows your mobile app and online store to accept payments using Apple Watch, iPhone, iPad, or Safari.
 
 ## Getting Started
 
-If this is your first time implementing our Payments API we recommend reviewing our [Payments API reference guides](https://dev.na.bambora.com/docs/references/merchant_API/v1-0-3/), to get familiar with Bambora's Payments API.
+If this is your first time implementing our Payment APIs we recommend reviewing our [Payment APIs reference guide](/docs/references/payment_APIs/), to get familiar with Bambora's Payments API's.
```
Comment (davidlight77) [inline source/docs/guides/apple_pay/index.md]: ...with our Payment APIs...

File: source/docs/references/payment_apis/index.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+  footer: na.tocs.na_nav_footer
+  toc: in-page
+  header_active: References
+
+---
+
+# Payment APIs
+Our Payment APIs are the RESTful interface to our payment gateway. It allows you to accept and analyze payments, create a payment profile to record customer details for future transactions, and create credit card tokens to reduce the scope of your PCI compliance.
+
+You can either interact with the API directly or through a SDK (a library of helper functions to simplify your integration with the SDK).
```
Comment (davidlight77) [inline source/docs/references/payment_apis/index.md]: ...an SDK...

**Pull request merged**